src/test/store/README.md

```diff
@@ -2,30 +2,30 @@
 
-# Reducers / Actions / Selectors
+## Reducers / Actions / Selectors
 
-For maintainability and making complex interactions sane, state management is broken up into different parts.
+For maintainability and making complex interactions sane, Redux state management is broken up into different parts.
 
  * *Reducers* care about how the canonical representation of the app state is stored. Actions come in, and a single state is retained.
- * *Actions* describe desired changes to the state, and provide the payload of information to be changed, but ignore how this is actually done.
- * *Selectors* provide a functional interface for selecting information within the data. They also map the data into derived states based on the canonical representation.
+ * *Actions* describe desired changes to the state, and provide the payload of information to do the changing, but ignore how this is actually done.
+ * *Selectors* provide a functional interface for selecting information within the data. They also map the data into derived states based on the canonical representation. These derivations are cached through memoization.
 
-# How to test the Redux store
+## How to test the Redux store
 
-The reducer, actions, and selectors each do a small part of the functionality of state management, and are decoupled so they can evolve separately over time. However perf.html's tests care about how the state management works together as a single unit. Redux store tests should use the same APIs that are being used by the rest of the application, and should not know about implementation details or the internals of reducers and actions. This differs from the canonical testing examples from the Redux documentation which assert that action creators produce specifically shaped actions, and reducers correctly reduce actions.
+The reducer, actions, and selectors each do a small part of the functionality of state management, and are decoupled so they can evolve separately over time. However perf.html's tests care about how the state management works together as a single unit. Redux store tests should use the same APIs that are being used by the rest of the application, and should not know about implementation details or the internals of reducers and actions. This differs from the canonical testing examples from the Redux documentation which individually assert that action creators produce specifically shaped actions, and reducers correctly reduce actions.
 
-# Testing steps
+## Testing steps
 
  1. Create a new blank store.
  2. Use a selector to grab the data that is being tested.
  3. Assert the initial state.
- 4. Dispatch an action to effect the desired change.
+ 4. Dispatch an action to affect the desired change.
```

Comment:
I think "effect" was correct. The change is what you want to happen. You don't want to modify the change.